src/plugins/signoff/types.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+export type SignoffState = {
+  resource: WorkflowInfo
```

Comment:
I'm surprised that this isn't declared as optional, as we're checking for it being possibly null in `src/plugins/signoff/components.js`.